### DISCUSSION TITLE
Queue search using MPD filters

### DISCUSSION THREAD

**Discussion by sevmonster** [jcorporation/myMPD#1012] [Ideas]
Potentially similar to #689, could we have the option in queue search fields to search using MPD filters? I see that [the API might already support this](https://github.com/jcorporation/myMPD/blob/d6b26c6fe37ee3d6fa8f2878040f4c11a1e2abf3/src/mympd_api/queue.c#L399) but the UI doesn't, unless I'm blind. For example, I want to remove all songs in the queue matching a specific word, but I cannot do that currently and have to remove them one by one (or use `mpc` instead)

**Comment by jcorporation** [comment]
myMPD enables the queue search supporting mpd filters only for mpd 0.24.x

**Reply by sevmonster** [reply]
You know, I read the part where you mentioned 0.24 in the documentation, but I didn't consider the fact that I did not have 0.24. It hasn't landed on Alpine yet apparently :)

**Reply by jcorporation** [reply]
MPD 0.24.x is not yet released.
